exercises/robot-name/description.md

```diff
@@ -8,6 +8,5 @@ which means that their name gets wiped. The next time you ask, it will
 respond with a new random name.
 
 The names must be random: they should not follow a predictable sequence.
-Random names means a risk of collisions. Your solution should not allow
-the use of the same name twice when avoidable. In some exercism language
-tracks there are tests to ensure that the same name is never used twice.
+Random names means a risk of collisions. Your solution should ensure that
```

Comment:
I was wondering if we should replace "should" with something more strong, like "must".